spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/properties/AwsCredentialsProperties.java

```diff
@@ -93,4 +96,55 @@ public void setProfilePath(String profilePath) {
 		this.profilePath = profilePath;
 	}
 
+	public StsCredentialsProperties getSts() {
+		return sts;
+	}
+
+	public void setSts(StsCredentialsProperties sts) {
+		this.sts = sts;
+	}
+
+	public static class StsCredentialsProperties {
+
+		/**
+		 * The arn of the role to be assumed.
+		 */
+		private String roleArn;
```

Comment:
@maciejwalkowiak WDYT about dropping `role` prefix for all the attributes in this class? Looks redundant to me